### DISCUSSION TITLE
No Logs for Web Console

### DISCUSSION THREAD

**Discussion by gareththered** [TechnitiumSoftware/DnsServer#692] [General]
I'm trying to get to the bottom of why one of my Technitium DNS servers doesn't allow me to connect to the Web Console over a HTTPS connection (port 53443) when using a CA provided certificate, whereas the other DNS server does allow me to connect over HTTPS, even though both servers have certificates issued by the same CA.  I'm struggling because there is no information in the logs whatsoever.

I know networking/routing is fine as it connects if the Web Console is using a self-signed certificate.  I also noticed that if I use `gnutls-cli <host>:53443` the server sends two certificates in the chain on the working DNS, but only a single certificate from the failing server.  Checking the PFX files with `openssl pkcs12 -in <file> -nokeys` shows all the certificates are there.

As there are no logs (or as I can't find the logs?) I'm now at a loss as to why Technitium can't find the rest of the certificate chain from the PFX.  Any thoughts?

**Comment by ShreyasZare** [comment]
Thanks for asking. This is a known issue and you can read more on it [here](https://github.com/TechnitiumSoftware/DnsServer/issues/610). The issue is already fixed and the update for it will be available mostly by coming weekend. This update should fix the issue that you are facing.

**Comment by gareththered** [comment]
Thank you, and apologies for the noise.  I should have carried out more research!

**Reply by ShreyasZare** [reply]
You're welcome. Do create an issue if the update when available does not fix the issue you have.

**Comment by ShreyasZare** [comment]
Technitium DNS Server v11.4 is now available that should fix the issue you mention. Do update and let me know your feedback.

**Comment by gareththered** [comment]
Apologies for the slow response.  I've just upgraded to 11.4 and it now works.

**Reply by ShreyasZare** [reply]
Thanks for letting me know.
